pkg/client/src/app/pages/applications/analysis-wizard/components/select-card.css

```diff
@@ -0,0 +1,3 @@
+.select-card__component__empty-state {
+  padding: 0 !important;
```

Comment:
Any reason we aren't using the spacing library here from pf?

Reply:
That's legacy code from MTA. 
We can replace that.